src/main/java/duke/operations/TaskList.java

```diff
@@ -0,0 +1,129 @@
+package duke.operations;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Represents a TaskList of Duke. A <code>TaskList</code> corresponds to
+ * containing the tasklist operations. e.g., it has operations to add/delete tasks in the list.
+ */
+public class TaskList {
+    protected static final ArrayList<Task> TASK_ARRAY_LIST = new ArrayList<>();
+
+    public int getTaskArrayListSize() {
+        return TASK_ARRAY_LIST.size();
+    }
+
+    /**
+     * Prints the total number of tasks.
+     *
+     * @param num total number of tasks.
+     * @return the String to be printed.
+     */
+    public String totalTasks(int num) {
```

Comment:
From what i notice, ur num in simply the size of the arraylist which u can get but calling getTaskArrayListSize() or TASK_ARRAY_LIST.size(). Why did you input it as an argument?